src/Core/src/Primitives/ToolTip.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Maui
+{
+	public partial class ToolTip
+	{
+		/// <summary>
+		/// ToolTip content.
+		/// </summary>
+		public object? Content { get; set; }
```

Comment:
It's not `string? Text` to open the API in the future to allow any content? Although, with another implementation we can achieve similar results by allowing richer content, the default native APIs expect a string.

https://developer.apple.com/documentation/uikit/uicontrol/3750832-tooltip
https://developer.android.com/reference/androidx/appcompat/widget/TooltipCompat

Reply:
I changed it to `ToolTipProperty.Text` at the controls layer. Since we only support `Text` that seems the least surprising. If we start supporting `Content` then we can just add the content property.